cpp/src/ray/worker/default_worker_experience.cc

```diff
@@ -0,0 +1,90 @@
+#define BOOST_BIND_NO_PLACEHOLDERS
```

Comment:
Why do we need the default_worker_experience.cc? I like this code as an example BTW.

Reply:
The test code and default_worker should be one binary now.